S/SLEEF/build_tarballs.jl

```diff
@@ -0,0 +1,70 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SLEEF"
+version = v"3.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/shibatch/sleef.git", "8df2bce4596ec6b86f357beebf2e428564f0a4ec"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/sleef
+mkdir build-native
+cd build-native
+PATH=`echo $PATH | cut -d":" -f 3-` \
+    CC=$HOSTCC \
+    CXX=$HOSTCXX \
```

Comment:
You may want to use `-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}`

Reply:
Indeed.